cypress/support/stackable/util.js

```diff
@@ -112,3 +120,27 @@ export function getActiveTab( callback = () => {} ) {
 			callback( tab )
 		} )
 }
+
+/**
+ * Function for checking if text exists within a DOM element in a recursive manner
+ *
+ * @param {jQuery} $parentElement DOM element to be searched
+ * @param {string} textToMatch text being searched in element
+ * @return {boolean} true if it contains text to be matched and false if it doesn't
+ */
+export function elementContainsText( $parentElement = Cypress.$( 'body' ), textToMatch = '' ) {
+	function _compareTextsRecursive( $parentElement ) {
```

Comment:
although the `$parentElement` variable inside `_compareTextsRecursive` is scoped, let's change the variable name to avoid confusions/conflict with the outer `$parentElement` variable

Reply:
Got it! I'll rename it to just element